src/main/java/bettercalculator/BetterCalculator.java

```diff
@@ -0,0 +1,61 @@
+package bettercalculator;
+
+interface BetterCalculator {
```

Comment:
What interface in this place did here? Do you see any benefit of that? I don't.  Still, this file contains all methods with their implementation.

All these methods are similar. They accept 2 double arguments and return 1 result (type double). Don't you think that there is a better place to use the interface?